rest/resource-server/src/docs/asciidoc/schedule.adoc

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright Siemens AG, 2023. Part of the SW360 Portal Project.
+//
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+
+[[resources-schedule]]
+=== Schedule
+
+The Schedule resource is used to get and list the Schedule requests.
+
+[[unschedule-services]]
+==== cancel all schedule.
```

Comment:
replace it with Cancel all scheduled services.

Reply:
done.